### ISSUE TITLE
How to do "train_test_split" for image data set used in pyrasterframes?

### ISSUE THREAD

**Issue by JenniferYingyiWu2020** [developmentseed/sat-ml-training#59]
Hi,
    I have got the code for "supervised machine learning" of pyrasterframes, and the link is "https://rasterframes.io/supervised-learning.html". I have seen the author used 12 ".tiff" to train the machine learning model, however the training set and testing set has not been split. Also, I used the code "x_training_data, x_test_data, y_training_data, y_test_data = train_test_split(x, y, test_size = 0.3)" and hoped to split the training set and testing, but I am not sure which one is "x" and "y". So, could you pls help to give me some suggestions on how to split the training set and testing set for the program on the  the link "https://rasterframes.io/supervised-learning.html"? Thanks!
![1](https://user-images.githubusercontent.com/72843232/106408083-a3175300-6478-11eb-9a07-c0e7960eb6ab.png)
![2](https://user-images.githubusercontent.com/72843232/106408094-a9a5ca80-6478-11eb-8315-9200557f2a5f.png)
![3](https://user-images.githubusercontent.com/72843232/106408097-add1e800-6478-11eb-83f6-61862a91b595.png)
![4](https://user-images.githubusercontent.com/72843232/106408107-b0ccd880-6478-11eb-9388-51378ba591f5.png)

**Comment by JenniferYingyiWu2020** [comment]
Hi,
    I have add the "unsupervised machine learning" and "supervised machine learning" under my github repository "https://github.com/JenniferYingyiWu2020/rasterframes-GeoTIFFs/tree/main/machine-learning". What I needed is to do "train_test_split" for the image data set of "unsupervised machine learning" and "supervised machine learning". So, could you pls give me some suggestions? Thanks!
![6](https://user-images.githubusercontent.com/72843232/106409043-a7dd0680-647a-11eb-8a27-d051e3c3c5c6.png)

**Comment by JenniferYingyiWu2020** [comment]
Hi,
    I have read your codes about "supervised machine learning" (https://rasterframes.io/supervised-learning.html), and found out that the multi-band image is "['B01', 'B02', 'B03', 'B04', 'B05', 'B06', 'B07', 'B08', 'B09', 'B11', 'B12']". Now, I need to split "model_input" as training_set and test_set, however the "model_input" is a DataFrame, also the "['B01', 'B02', 'B03', 'B04', 'B05', 'B06', 'B07', 'B08', 'B09', 'B11', 'B12']" is Tile Array. So, could you pls give me some suggestions on how to convert the "model_input" as a multi-matrix? If the "model_input" can be converted to a multi-matrix, then the function "train_test_split" can be used to split the multi-band image as a training set and a test set. Thanks!

**Comment by wildintellect** [comment]
This project is not related to rasterframes.io
